Yoast/Tests/Namespaces/NamespaceDeclarationUnitTest.2.inc

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace; // Error - no name.
+
+namespace {} // Error x 3 - no name + scoped namespace declaration + not the only namespace declaration in the file.
```

Comment:
Would these cases be relevant?

```php
namespace /* some comment */ {}
```

or
```php
namespace /* some comment */;
```

or 
```php
/* some comment */ namespace;
```

Reply:
The first two: yes. The sniff already takes this into account, but never a bad thing to document this in the unit tests. :+1: 

The third: no. The sniff only looks at what's after the `namespace` keyword, not what's before, so the third wouldn't be testing anything.

I'll add those two.